jenkins/run

```diff
@@ -156,7 +157,7 @@ ${PODMAN} run \
     --rm \
     ${EXTRA_PODMAN_SWITCHES} \
     ${_image} \
-    jenkins/runner "${@}"
+    "${@}"
```

Comment:
Was there a reason we used `runner` to force a `bash` parent instead of using `--init` before?

Reply:
I believe we, meaning me, didn't know about the `--init` switch.